extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -134,4 +134,19 @@ public function hasPageApprovedId( $wikiId, $pageId, $oldid ) {
 		}
 		return false;
 	}
+
+	/**
+	 * Link for adding new section on script talk page. Prefilled with standard explanation of rejection.
+	 * @param \Title $title Title object of JS page
+	 * @return string full link to edit page
+	 */
+	public function prepareProvideFeedbackLink( \Title $title ) {
+		$params = [
+			'action' => 'edit',
+			'section' => 'new',
+			'useMessage' => 'content-review-rejection-explanation'
+		];
+		$talkPage = $title->getTalkPage();
```

Comment:
You could skip setting the Talk Page object to a variable and just return the URL, what do you think?